### PR TITLE
Add cookie policy module

### PR DESCRIPTION
**Pull request by anthonydillon**
# Done
- Add cookie policy
- Quick fix links in a `strip-dark`

## QA
- Check code as the cookie policy is added with JS it is unavailable in the demo

### CODE REVIEW

File: scss/modules/_cookie-policy.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+/// @since        0.0.21
+////
+
+/// cookie policy
+/// @group Utils
+@mixin cloud-cookie-policy {
+  $cookie-background: #fae4dc;
+
+  .cookie-policy {
+    @include box_shadow(0 -1px 2px rgba(0, 0, 0, .2));
```
Comment (richmccartney) [inline scss/modules/_cookie-policy.scss]: @anthonydillon Might want to remove this mixin and check if there are any others as it won't build.

File: scss/modules/_cookie-policy.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+////
+/// @author       Juju GUI team at Canonical Ltd
+/// @link         http://ubuntudesign.github.io/cloud-vanilla-theme/docs/#cookie-policy
+/// @since        0.0.21
+////
+
+/// cookie policy
+/// @group Utils
+@mixin cloud-cookie-policy {
+  $cookie-background: #fae4dc;
```
Comment (richmccartney) [inline scss/modules/_cookie-policy.scss]: @anthonydillon we should move this into global settings I feel.

File: scss/modules/_cookie-policy.scss [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      padding: 0;
+      text-decoration: none;
+      text-indent: -9999px;
+      width: 16px;
+    }
+  }
+
+  html.no-svg,
+  html.opera-mini {
+    .cookie-policy .link-cta {
```
Comment (richmccartney) [inline scss/modules/_cookie-policy.scss]: @anthonydillon can we remove the ```html``` element from this with out it breaking? We shouldn't tie ourselves to any element just incase people put it on ```body``` or something else for some reason.

**Comment by richmccartney** [comment]
@anthonydillon I made a few comments :)

**Comment by anthonydillon** [comment]
@richmccartney thank you very much. Great spots! I have added the cookie policy mark up to the demo too so you can check it looks ok.

**Comment by richmccartney** [comment]
@anthonydillon looks good :)

**Pull request merged**